classes/form/CustomerForm.php

```diff
@@ -138,6 +138,15 @@ public function validate()
             );
             $birthdayField->setValue($dateBuilt->format('Y-m-d'));
         }
+
+        $passwordField = $this->getField('password');
+        if(!Validate::isPasswd($passwordField->getValue())) {
+            $passwordField->AddError($this->translator->trans(
+                'The password should be at least 5 characters long',
+                [],
+                'Shop.Notifications.Error'
+            ));
+        }
```

Comment:
The error message will be incorrect if the password contains more than 72 characters
https://github.com/202ecommerce/PrestaShop/blob/fix-password-validation/classes/Validate.php#L541

**Suggestion :** 

Return a new common error message for both cases (<5 and > 72)  :  
Example :  `Password must be between 5 and 72 characters long` or something like that

Reply:
I hadn't thought about it, I'll fix it

Reply:
ping @LouiseBonnard @Junebyun if you need help with this!

Thank you @florian-202